src/app/app-routing.module.ts

```diff
@@ -8,17 +8,19 @@ import {GraphComponent} from './_shared/graph/graph.component';
 
 import {TaskListViewComponent} from './_shared';
 import {GraphViewComponent} from './_shared/tasks/graph-view/graph-view.component';
+import {VolumesComponent} from './_shared/volumes/volumes.component';
 import {TaskCardViewComponent} from './_shared/tasks/card-view/task-card-view.component';
 import {ServicesCardViewComponent} from './_shared/services/card-view/services-card-view.component';
 
-
 const routes: Routes = [
     {path: 'containers', component: ContainersComponent},
     {path: 'listServices', component: ServiceListViewComponent},
     {path: 'networks', loadChildren: 'app/_shared/networks/network.module#NetworkModule'},
+    {path: 'volumes', loadChildren: 'app/_shared/volumes/volumes.module#VolumesModule'},
     {path: 'graph', component: GraphComponent},
     {path: 'graphTasks', component: GraphViewComponent},
     {path: 'listTasks', component: TaskListViewComponent},
+    {path: 'volumes', component: VolumesComponent},
```

Comment:
Why are there 2 volume paths? (It could be the loadchildren thing that I don't understand) 😅

Reply:
Fixed in https://github.com/TripleParity/docks-ui/commit/35c5044c1ffeee6e375954c52a31e9f9cb7ef5ef

Nice, probably didn't pick it up as routing would still have used the lazy loading instead but regardless was a problem.